src/control/cmd/daos_server/main.go

```diff
@@ -110,7 +108,8 @@ func parseOpts(args []string, opts *mainOpts, log *logging.LeveledLogger) error
 				}
 			}
 		} else if opts.ConfigPath != "" {
-			return errors.Errorf("DAOS Server config has been supplied but this command will not use it")
+			return errors.Errorf("DAOS Server config filepath has been supplied but " +
+				"this command will not use it")
```

Comment:
Why add this? We don't have line limits on control plane code and this just adds to review noise.

Reply:
apologies, agree was unnecessary